tools/genman/gen_oadm_man.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/openshift/origin/pkg/cmd/admin"
+	"k8s.io/kubernetes/cmd/genutils"
+)
+
+func GenOadmMan(cmdName string) {
+	// use os.Args instead of "flags" because "flags" will mess up the man pages!
+	path := "docs/man/" + cmdName
```

Comment:
All 3 functions(GenOadmMan, GenOcMan, GenOpenshiftMan) I see exact same code, can we just write one common function to generate the markdown and call with commandname to it.?